Files/Interacts/Interaction.cs

```diff
@@ -842,6 +842,19 @@ public async void PasteItem_ClickAsync(object sender, RoutedEventArgs e)
 
         public async Task PasteItems(DataPackageView packageView, string destinationPath, DataPackageOperation acceptedOperation)
         {
+            if (!packageView.Contains(StandardDataFormats.StorageItems))
```

Comment:
Note: this change is not specific to pasting to the recycle bin but can happen when pasting in any folder

Reply:
Good catch